src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/get-started/opentelemetry-tutorial-python.mdx

```diff
@@ -40,47 +40,61 @@ import opentelemetryPythonLogCapturedinTraceSpan from 'images/opentelemetry_scre
-
-    * [Tutorial 1: Run the pre-instrumented demo app:](#pre-instrum) This is the fastest way to send some demo data to New Relic and see how it is displayed in the UI. In this tutorial, the demo app has pre-loaded instrumentation and SDK configurations that follow our [best practices](/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/best-practices/opentelemetry-best-practices-overview/) to generate and export metrics, logs, and traces. You can inspect our code and apply relevant sections to your own apps.
-    * [Tutorial 2: Monitor the demo app with the OpenTelemetry Python agent:](#python-agent) Instead of using our pre-instrumented demo app, you can use the OpenTelemetry Python agent to automatically monitor our demo app. You don't need to open the demo code and make any changes&mdash;just install and run the agent alongside the app.
-    * [Tutorial 3: Set up the demo app manually:](#manual-instrum) In this track, you'll roll up your sleeves and tinker with the engine of the car. This is the approach to take if you want to have the most control over what telemetry is reported and want to see details about how it's done. You'll manually insert instrumentation into our demo app to capture telemetry and you'll configure the SDK to export that data to New Relic.
+<img
+  title="Screenshot showing response time, throughput, and error rate"
+  alt="Screenshot showing response time, throughput, and error rate"
+  src={openttelemetryPythonIntroImage}
+/>
 
-<Callout variant="tip">
-You have two choices for exporting data from your application to New Relic via OTLP:
-* Directly from your app
-* Via an OpenTelemetry collector
+<figcaption>
+  After you finish any of these tutorials, you can view span metrics in charts like these.
+</figcaption>
 
-This guide covers the first option. If you wish to export your data via a collector, check out this [collector documentation](/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/collector/opentelemetry-collector-basic/) for details.
-</Callout>
 ## Requirements [#requirements]
 
 Before you get started, make sure you have the following:
 
 * A New Relic account: Sign up for a [free account](https://newrelic.com/signup) if you don't already have one.
 * Python 3.8 or higher, and pip (included in Python version 3.4 and later): If you don't already have these, see these [download options](https://www.python.org/downloads/).
 
-## Tutorial 1: Run the pre-instrumented demo app [#pre-instrum]
+## Tutorials [#tutorials]
+
+Although each tutorial uses the same demo app, they have different approaches to help you become acquainted with OpenTelemetry and New Relic.
+
+* [Pre-instrumented app](#pre-instrumented-tutorial): Run our pre-instrumented demo app for a quick way to see data in our UI.
+* [OpenTelemetry Python agent](#python-agent-tutorial): Monitor our demo app with the OpenTelemetry Python agent.
```

Comment:
What is the set up method for the 2nd option? Don't these all technically monitor a demo app with the Otel Python agent? Is it using a different install that isn't manual or pre-instrumented?

Reply:
I'm not sure. The doc itself is titled 'python' so I assumed all of these methods technically use Python OTel in some capacity. The other two tabs reference _how_ they get data -- either using a pre-instrumented app (is it with the Python agent, like the doc title implies?)  or manually installing _something_, but the middle tab is only titled by the noun. Nothing to indicate how instrumentation is happening. If i'm understanding:

- Tab 1 just lets users view data without any instrumentation
- Tab 2 walks users through instrumenting a demo app, without custom instrumentation. They still manually install libararies/agent/whathaveyou
- Tab 3 introduces custom instrumentation to the demo app, but is also a manual procedure

maybe the titles could imply the kind of data they get, rather than how they're instrumented.